.gitignore

```diff
@@ -11,3 +11,6 @@ venv*/
 .python-version
 build/
 dist/
+.idea/
```

Comment:
Those ignores are not going to be accepted :sweat_smile:

Reply:
I tried: https://github.com/encode/uvicorn/pull/1001

Reply:
ahahaha :D

Reply:
Done